merkle_tree_wasm/Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+name = "merkle_tree_wasm"
```

```suggestion
name = "merkle-tree-wasm"
```

The wasm-pkg build step takes the name of the crate to set the name of the pkg. So lets put `-` instead of `_` to stay consistent across all the repo.